templates/SilverStripe/Forms/SelectionGroup.ss

```diff
@@ -13,7 +13,7 @@
     <ul class="{$ExtraClass.ATT}">
         <% loop $FieldSet %>
             <li {$Selected}>
-                {$RadioButton} {$RadioLabel}
+                <label for="{$Up.ID}_{$Pos}">{$RadioButton} {$RadioLabel}</label>
```

Comment:
If `$RadioButton` renders an input and it's inside the label element, then we don't need the `for` attribute.

See `SelectionGroup.ss` in framework: `<label>{$RadioButton} {$RadioLabel}</label>`